### ISSUE TITLE
alarm_control_panel icon doen't show

### ISSUE THREAD

**Issue by FaBRiK74** [jcwillox/lovelace-paper-buttons-row#45]
The alarm_control_panel entity has a changing icon based on state. When I add the alarm_control_panel entity under "buttons:", the icon disappears. If I put the same code right under "entities:" it works normally. See the attached screenshot. On the upper side there is a normal entity glance card with the same entity "Riasztó" and on the bottom the missing icon in Paper Buttons Row with the same code below.
![alarm icon](https://user-images.githubusercontent.com/78682753/114823544-f59fb480-9dc3-11eb-8135-39fe67ddaef0.jpg)

And the code:
```yaml
type: entities
entities:
  - type: 'custom:paper-buttons-row'
    buttons:
      - entity: alarm_control_panel.riaszto
        name: Riasztó
        state_color: true
        tap_action:
          action: navigate
          navigation_path: biztonsag
```

**Comment by FaBRiK74** [comment]
Sorry it is changed the code. The real lines are:
![code](https://user-images.githubusercontent.com/78682753/114823766-3f889a80-9dc4-11eb-95ad-4341730c5d13.jpg)

**Comment by FaBRiK74** [comment]
Unfortunately it doesn't solve the problem. The icon still missing.
thanks for the code quoting tip.

**Comment by weppa** [comment]
@FaBRiK74 I have the same problem with sensor entities not showing the battery icon based on it's state, so have manually overriden the icon (for now).

**Comment by jcwillox** [comment]
Ahh I think I see the issue, I only have default icons for these domains https://github.com/jcwillox/lovelace-paper-buttons-row/blob/2333d54edf7985a0c625727eff980305efcbbbcd/src/entity.js#L24-L33
Otherwise, it will try and extract the icon from `entity.attributes.icon`, if it doesn't find one, and you don't manually specify it, then it wont show up.

I'm planning some larger changes to `paper-buttons-row`, which will include switching to using an external library for functionality like that, which should fix this issue.

**Comment by jcwillox** [comment]
This has been solved in [1.0.0](https://github.com/jcwillox/lovelace-paper-buttons-row/releases/tag/1.0.0), I've started using `custom-card-helpers` which offers much more advanced icon extraction.

**Comment by weppa** [comment]
Legend! Thanks for fix and keep up the great work.
